tests/test_update_file_manager.py

```diff
@@ -1,6 +1,12 @@
 """Unit test for modules/update_manager/update_manager.py"""
 
 from tests.module_factory import ModuleFactory
+import json
+import requests
+import pytest
+import time
+import os
+from unittest.mock import Mock, mock_open
 
 
 def test_getting_header_fields(mocker, mock_db):
```

Comment:
we should call it test_get_e_tag() , we should stick to having the original function's name in the test functions name as much as possible